### PR TITLE
Continuous function InnerSimplified

### PR DESCRIPTION
**Pull request by ryyanmapes**
This is for https://github.com/asc-community/AngouriMath/issues/428 , but I have only done this for continuous functions because it's sort of unclear.

### CODE REVIEW

**Comment by WhiteBlackGoose** [comment]
What's unclear?

**Comment by ryyanmapes** [comment]
Should InnerSimplified be added only to continuous functions like in the example in the issue, or should they also be extended to logic and all areas?

**Comment by WhiteBlackGoose** [comment]
Yeah, should be applied to logic and other functions too. If the user wants to run a function which doesn't simplify anything implicitly, they will just use functions from `MathS`.

File: Sources/Wrappers/AngouriMath.FSharp/Functions.fs [hunk 1]
```diff
@@ -29,59 +29,59 @@ let latex x =
...
-let substituted x value expr = (parsed expr).Substitute(parsed x, parsed value)
+let substituted x value expr = (parsed expr).Substitute(parsed x, parsed value).InnerSimplified
 
 /// Returns a multiline string representation of matrix
 let printedMatrix (x: AngouriMath.Entity.Matrix) = x.ToString(true)
 
 /// Returns a matrix modified according to the modifier
 let modifiedMatrix (x: AngouriMath.Entity.Matrix) m =
-    x.With(new System.Func<int, int, AngouriMath.Entity, AngouriMath.Entity>(m))
+    x.With(new System.Func<int, int, AngouriMath.Entity, AngouriMath.Entity>(m)).InnerSimplified
```
Comment (WhiteBlackGoose) [inline Sources/Wrappers/AngouriMath.FSharp/Functions.fs:39]: You need to return Matrix here. Use `.Downcast<Entity.Matrix>()` extension from Honk# or downcast via F#'s syntax. F# syntax would look like
```fs
let myFunction ... : Entity.Matrix = downcast ...
```

File: Sources/Wrappers/AngouriMath.FSharp/Functions.fs [hunk 2]
```diff
@@ -29,59 +29,59 @@ let latex x =
...
 
 /// Gets the transposed form of a matrix or vector
-let transposed (m: AngouriMath.Entity.Matrix) = m.T
+let transposed (m: AngouriMath.Entity.Matrix) = m.T.InnerSimplified
```
Comment (WhiteBlackGoose) [inline Sources/Wrappers/AngouriMath.FSharp/Functions.fs:42]: Same here

File: Sources/Wrappers/AngouriMath.FSharp/Functions.fs [hunk 3]
```diff
@@ -267,8 +267,8 @@ let limitSidedNode x destination side expr =
 /// provided the statement or equality
 let solutions x expr =
     match parsed expr with
-    | :? Entity.Statement as statement -> statement.Solve(symbol x)
-    | func -> (equality func 0).Solve(symbol x)
+    | :? Entity.Statement as statement -> statement.Solve(symbol x).InnerSimplified
+    | func -> (equality func 0).Solve(symbol x).InnerSimplified
```
Comment (WhiteBlackGoose) [inline Sources/Wrappers/AngouriMath.FSharp/Functions.fs:271]: Same here, we want to get `Set` as a result

File: Sources/Wrappers/AngouriMath.FSharp/Functions.fs (hunk 1 above)
Comment (Happypig375) [reply Sources/Wrappers/AngouriMath.FSharp/Functions.fs:39]: Stop being weird. Use `:?> Entity.Matrix`.

**Review by WhiteBlackGoose — APPROVED** [review]
Looks good

**Comment by WhiteBlackGoose** [comment]
Thanks for your contribution!

**Pull request merged**